x-pack/platform/plugins/shared/alerting/server/application/rule/methods/mute_alert/mute_instance.ts

```diff
@@ -19,7 +19,7 @@ import { updateMeta } from '../../../../rules_client/lib';
 
 export async function muteInstance(
```

Comment:
I didn't look at "unmute", but I'm guessing it's not checking either.  Should we do the same thing with that API?